project/tikv/project.json

```diff
@@ -0,0 +1,111 @@
+{
+    "_comment": [
+        "Items of lists in this file are in alphabetical order. Please keep it on modification."
+    ],
+
+    "maintainers": [
```

Comment:
just curious how the maintainers are decided.

Reply:
According to https://github.com/tikv/community/issues/118 it writes

> Where do current things go?
> Here is the migration details from current SIG membership to project membership.
> migration
> sig-scheduling migrates to PD project
> sig-raft migrates to Raft project
> sig-engine / sig-coprocessor / sig-transaction migrate to TiKV project
> techLeaders and coLeaders migrate to maintainers 

and later @BusyJay proposed https://github.com/tikv/community/pull/122#discussion_r646342537